src/main/java/uk/gov/pay/api/service/ConnectorService.java

```diff
@@ -82,4 +85,17 @@ public RefundFromConnector getPaymentRefund(String accountId, String paymentId,
 
         throw new GetRefundException(connectorResponse);
     }
+
+    public SearchRefundsResponseFromConnector searchRefunds(Account account, Map<String, String> queryParams) {
+        Response connectorResponse = client
+                .target(connectorUriGenerator.refundsURIWithParams(account, queryParams))
+                .request()
+                .accept(MediaType.APPLICATION_JSON)
+                .get();
+
+        if (connectorResponse.getStatus() == SC_OK) {
+            return connectorResponse.readEntity(SearchRefundsResponseFromConnector.class);
```

Comment:
think you've removed the `try/catch` for `ProcessingException` here?

Reply:
Good catch! Updated here https://github.com/alphagov/pay-publicapi/pull/654/commits/ad9beb687556b75d7f754f129e5fec70cd356b6a#diff-07acbd73c1ac179bb38becfe7d2de74bR98